servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt

```diff
@@ -32,13 +32,17 @@ import org.springframework.web.reactive.function.server.html
 @Configuration
 class PlaygroundRouteConfiguration(
     private val config: GraphQLConfigurationProperties,
-    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource
+    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource,
+    @Value("spring.webflux.base-path") private val contextPath: String?
```

Comment:
does this actually work? I think it should be `@Value("\${spring.webflux.base-path}")`

Reply:
Yes, you are right. I will update it.